#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# n
"""
Traceback (most recent call last):
  File "error.py", line 3, in <module>
    n
NameError: name 'n' is not defined
"""
